reana_commons/utils.py

```diff
@@ -284,10 +282,10 @@ def get_disk_usage(
 
     :return: List of dicts with file name and size.
     """
-    reana_fs = fs.open_fs(SHARED_VOLUME_PATH)
-    if not reana_fs.exists(directory):
+    # reana_fs = fs.open_fs(SHARED_VOLUME_PATH)
+    if not os.path.exists(directory):
         raise REANAMissingWorkspaceError("Directory does not exist.")
-    absolute_path = reana_fs.getospath(directory)
+    absolute_path = directory
```

Comment:
Very minor: we could get rid of `absolute_path` and use `directory` instead, WDYT?

Reply:
Yes, it is better!